src/shared/components/oncoprint/ResultsViewOncoprint.tsx

```diff
@@ -201,6 +201,31 @@ export default class ResultsViewOncoprint extends React.Component<IResultsViewOn
             }
         );
 
+        this.putativeDriverSettingsReaction = reaction(
```

Comment:
as above, i think we should really try to avoid making reactions unless it's really necessary.  could this not be accomplished with a computed getters?   i think the setting should be immutable.

Reply:
this is to maintain the relationship between distinguishDrivers and any of the mutationAnnotationSettings being set to true. I think this is a clean way to do it because the mutationAnnotationSettings can be modified from here or from the store.